_content/lorawan/security.md

```diff
@@ -3,7 +3,7 @@ title: Security
 section: Specifications
 zindex: 2
 redirect_from:
- - current//#security/
+ - ./index/#security/
  - current/security/
```

Comment:
I still don't get which redirects these are intended to be. Can you specify the existing sources that are being redirected here?

Reply:
I am not sure about the `current` maybe I should replace it with `lorawan`.
line 6 is to redirect from the sub-part `security` in `index.md`.

Reply:
https://github.com/jekyll/jekyll-redirect-from#usage